docs/docs/code/python/README.md

```diff
@@ -0,0 +1,279 @@
+# Writing Python in Code Steps
+
+**Anything you can do in Python can be done in a Pipedream Workflow**. This includes using any of the [350,000+ PyPi packages available](https://pypi.org/) in your Python powered workflows. 
+
+Pipedream supports [Python v{{$site.themeConfig.PYTHON_VERSION}}](https://www.python.org) in workflows.
+
+::: warning
+Python steps are available in a limited alpha release.
+
+You can still run arbitrary python code, including [sharing data between steps](/code/python/#sharing-data-between-steps) as well as [accessing environment variables](/code/python/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
+:::
+
+
+## Adding a Python code step
+
+1. Click the + icon to add a new step
+2. Click "Custom Code"
+3. In the new step, select the Python language runtime in language dropdown
+
+## Logging & debugging
+
+You can use `print` at any time in a Python code step to view information as the script is running.
+
+The output for the `print` logs will appear in the `Results` section just beneath the code editor.
+
+<div>
+<img alt="Python print log output in the results" src="./images/print-logs.png">
+</div>
+
+## Using 3rd party packages
+
+You can use any packages from [PyPi](https://pypi.org) in your Pipedream workflows. This includes popular choices such as:
+
+* `requests` for making HTTP requests
```

Comment:
Worth linking to packages?

Reply:
I was debating that, I wanted a monospace font for the package name, didn't want the link styling to override that. Let me try it out thought.

Reply:
Format looks good. I'll link all package 👍